### PR TITLE
Moved from Ant to Maven

### PR DESCRIPTION
**Pull request by Kathrin-Huber**
replaced build.xml with pom.xml and brought folder structure to a maven compatible form.

### CODE REVIEW

File: Kitodo/pom.xml [hunk 1]
```diff
@@ -30,7 +29,7 @@
             <artifactId>log4j</artifactId>
             <version>1.2.16</version>
         </dependency>
-        <!-- UGH - Libraray -->
+        <!-- UGH - Libary -->
```
Comment (stweil) [inline Kitodo/pom.xml]: Still can be improved. :-)

File: README.md [hunk 2]
```diff
@@ -17,27 +17,20 @@ Building
...
 
 The project structure is IDE independent. The tool for issue building, testing
-and packaging of the application is [Ant](http://ant.apache.org/). Ant gets
-configured by providing a build.properties file next to the build.xml build-file.
+and packaging of the application is [Maven](https://maven.apache.org/).
 
 ### Dependencies
 
-All dependent libraries are shipped with the source code. They can be found in the directory `/Goobi/WEB-INF/lib/`.
+Available dependencies are fetched from Maven Central. Further dependencies(not available in Maven Central) are located in Kitodo\src\main\webapp\WEB-INF\lib
```
Comment (stweil) [inline README.md]: Please don't use Windows path separators. Use `/` instead.

There is a blank missing before `(`. Also missing is a terminating `.`.

File: README.md [hunk 3]
```diff
@@ -17,27 +17,20 @@ Building
...
-### Vanilla build using Ant
+### Vanilla build using Maven
 
-In order to build the application with (the rather useless) default configuration, you have to ensure that there is a Tomcat distribution ready. Then follow these simple steps in the project top-level directory:
-
-1. Use the example `build.properties.templates` file to create an actual `build.properties` file.
-	cp build.properties.template build.properties
-2. Edit the file and provide the path to your Tomcat distributions `lib/` directory in the `tomcat.dir.lib` parameter.
-3. Execute ant to generate Kitodo.Production distributable WAR and JAR files. All build artifacts (class files, javadoc, test results) will end up in the `build/` directory. All distributables end up in the `dist/` directory.
+execute "mvn clean package" and put generated war file in Tomcat.
```
Comment (stweil) [inline README.md]: Start with upper case: Execute ... instead of execute ...

**Comment by Kathrin-Huber** [comment]
Thanks for pointing out the misspellings, fixed them directly.

**Comment by stweil** [comment]
@Kathrin-Huber, thank you for addressing those issues. I suggest to close PR #571 and use your new PR as a starting point.

**Comment by matthias-ronge** [comment]
Technically, Eclipse integration works.

However, there is _one big drawback_: Obviously, eclips needs to create _two_ projects in the workspace to handle the Maven project, one for the repository root and one for the (each?) module in it. The project for the module is named like the “artifact ID” (_kitodo-production_). This leads to confusion, because the artifact ID and the file system and repository paths are distributed vice-verse. So if you check out the git repository and name the repository folder _kitodo-production_ (like the repository), this name is taken and _creating the Maven project fails_. Any other name works.

After having found out, I now have this setup which works:

_Path:_ https://github.kitodo/**kitodo-production**_/tree/branch-name/
_Folder:_ D:\Eclipse.WS\**kitodo**
_Project:_ **kitodo**
_Artifact ID:_ **kitodo**
_Path:_ https://github.kitodo/kitodo-production_/tree/branch-name/**Goobi**
_Folder:_ D:\Eclipse.WS\kitodo\**Goobi**
_Project:_ **kitodo-production**
_Artifact ID:_ **kitodo-production**

However, I would like to recommend a clean-up here. Example:

_Path:_ https://github.kitodo/**kitodo-production**_/tree/branch-name/
_Folder:_ D:\Eclipse.WS\**kitodo-production**
_Project:_ **kitodo-production**
_Artifact ID:_ **kitodo-production**
_Path:_ https://github.kitodo/kitodo-production_/tree/branch-name/**kitodo-webapp**
_Folder:_ D:\Eclipse.WS\kitodo-production\**kitodo-webapp**
_Project:_ **kitodo-webapp**
_Artifact ID:_ **kitodo-webapp**

**Comment by henning-gerhardt** [comment]
@matthias-ronge : This issue should be solved during modularization of the source code in this branch.

**Pull request merged**